docs/section4/4-5.md

```diff
@@ -0,0 +1,37 @@
+# ASHVAC – Rule 4-5
+
+**Rule ID:** 4-5
+ 
+**Rule Description:** HVAC fans shall remain on during occupied and unoccupied hours in spaces that have health and safety mandated minimum ventilation requirements during unoccupied hours.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 exception #2 Schedules for the proposed building  
+**Appendix G Section Reference:**  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** 
+
+1. P-PRM is modeled with a least one space with health and safety mandated minimum ventilation requirements during unoccupied hours. 
+
+**Manual Check:** None  
+**Evaluation Context:** Evaluate each data element  
+**Data Lookup:** None  
+**Function Call:** None
+
+## Rule Logic:
+- For each zone in P_RMR: `for zone_p in P_RMR...zones:`
+    - For each space in zone: `for space_p in zone_p.zones:`
+        - Check if space(s) is of type that has health and safety mandated minimum ventilation requirements during unoccupied hours: `if space_p.lighting_space_type in [CORRIDOR_HOSPITAL,LABORATORY_EXCEPT_IN_OR_AS_A_CLASSROOM,HEALTHCARE_FACILITY_OPERATING_ROOM]:`
+            - Set applicability flag: `rule_applicability_check = TRUE`
+                - For each terminal in zones that meet the applicabliity check: `for terminal_p in zone_p.terminals:`
+                    - Get HVAC system serving terminal: `hvac_p = terminal_p.served_by_heating_ventilation_air_conditioning_systems`
+                        - Get fan system of HVAC system: `fan_system_p = hvac_p.fan_systems`
+                        -   Get fan system operating schedule: `operating_schedule_p = fan_system_p.operating_schedule`
+
+  **Rule Assertion:**
+  - Case 1: For each applicable fan system the fan schedule is 1 for all hour: `if operating_schedule_p[i] == 1 for i in range(8760): PASS`
+
```

Comment:
I think the check for this RDS should be the same as for 4-3 and 4-4, to look at operation_during_occupied and operation_furing_unoccupied.

Reply:
Great, that is much simpler. Update made.